types/datum.go

```diff
@@ -1429,13 +1429,38 @@ func (d *Datum) convertToMysqlFloatYear(sc *stmtctx.StatementContext, target *Fi
 	return ret, err
 }
 
+func (d *Datum) convertStringToMysqlBit(sc *stmtctx.StatementContext) (uint64, error) {
+	bitStr, err := ParseBitStr(BinaryLiteral(d.b).ToString())
+	if err != nil {
+		// This is to be compatible with the previous processing method.
```

Comment:
please be more precise with this comment, because once the PR get merged, no one will know what the "previous processing method" is.